scripts/obtain/uninstall/debian-packages.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
Or we can have a single file `dotnet-uninstall.sh` which works for `OSX` and `Ubuntu`.